services/tenant-ui/frontend/src/store/contactsStore.ts

```diff
@@ -40,6 +40,16 @@ export const useContactsStore = defineStore('contacts', () => {
     contacts.value.filter((c) => c.state === CONNECTION_STATUSES.INVITATION)
   );
 
+  const findConnectionName = computed(() => {
+    return (connectionId: string) => {
+      // Find the connection alias for an ID
+      const connection = contacts.value?.find((c: any) => {
+        return c.connection_id === connectionId;
+      });
+      return connection && connection.alias ? connection.alias : '...';
```

Comment:
Is this returning the `...` for the UI? Would `undefined` be a better, more explicit option?

Reply:
Returning ... for loading state since the connection loading could be going while the element wanting to display it could be on there.
Could refactor for different logic for not found vs contacts loading
https://github.com/bcgov/traction/issues/576